_sass/custom/_home.scss

```diff
@@ -68,36 +68,25 @@
         margin: 1rem auto;
     }
 
-    .programs .col {
-        height: 80px;
-        align-content: center;
-        background: #fff;
-        margin: 4px;
-        padding: 0;
-        border-radius: 12px;
+    .programs article {
+        margin: 1rem .25rem;
     }
 
-    .programs .col a {
+    .programs article a {
         display: block;
-        position: relative;
-        width: 100%;
-        height: 100%;
-        align-content: center;
+        padding: 1.5rem 1rem;
+        background: #fff;
         border-radius: 12px;
-    }
 
-    .programs .col a::after {
```

Comment:
@sssoz is the 'whole rectangular area' still clickable with these styles removed? The whole CTA should be clickable and not just the title link text.